pulse-android-otel/pulse-upload-plugin/src/main/kotlin/com/pulse/upload/PulseUploadSourceMapsPlugin.kt

```diff
@@ -0,0 +1,321 @@
+package com.pulse.upload
+
+import java.io.File
+import java.net.HttpURLConnection
+import java.net.URL
+import org.gradle.api.DefaultTask
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.options.Option
+import org.gradle.api.tasks.TaskAction
+
+class PulseUploadSourceMapsPlugin : Plugin<Project> {
```

Comment:
```suggestion
class PulsePlugin : Plugin<Project> {
```
renaming as same plugin will be used in future for other tasks as well(we will not be creating different plugin for one individual task)

Reply:
Also move up this file `pulse/plugin/` folder